src/App.js

```diff
@@ -12,27 +12,19 @@ import Music from "./components/Music/Music";
 import Settigns from "./components/Settings/Settings";
 
 import "./App.css";
+import UsersContainer from "./components/Users/UsersContainer";
```

Comment:
Consider grouping this import with the Settings import - they belong to the same folder level

Reply:
done